### PR TITLE
Drones can no longer use guns and will die if on Central Command

### PR DESCRIPTION
**Pull request by ghost**
Sick and fucking tired of griefing drones, should help a little bit

yeah this nerfs clockdrones and syndrones but it's not like syndrones exist and clockdrones needed a nerf

if you really want i can make it so hacked drones can use guns, but  eh
#### Changelog

:cl:
rscadd: Drones can no longer use guns.
rscadd: Drones will die if on Central Command.
/:cl:

### CODE REVIEW

**Comment by Time-Green** [comment]
Drones don't have the NOGUN flag, I think it's human only

**Comment by Kmc2000** [comment]
Ehhhh syndrones are still used rarely, if you can exclude them, please do.

**Comment by Cruix** [comment]
NOGUNS is a species flag, you can't put it in the flags list of any mob.

**Comment by ghost** [comment]
@Cruix 

Well.

What do I do in it's place then

**Comment by Cruix** [comment]
`/mob/living/simple_animal/drone/can_use_guns(var/obj/item/weapon/gun/G)`
[tab]`src << "<span class='warning'>Optional message telling a drone that it is trying to break its laws.</span>`
[tab]`return FALSE`

File: code/modules/mob/living/simple_animal/friendly/drone/_drone.dm [hunk 1]
```diff
@@ -66,6 +66,12 @@
 	var/hacked = 0 //If we have laws to destroy the station
 	var/datum/personal_crafting/handcrafting
 
+/datum/mob/living/simple_animal/drone/spec_life(mob/living/simple_animal/drone = D)
+ 	if(D.stat == DEAD)
+ 		return
```
Comment (Super3222) [inline code/modules/mob/living/simple_animal/friendly/drone/_drone.dm]: This lines acting up again...

File: code/modules/mob/living/simple_animal/friendly/drone/_drone.dm [hunk 2]
```diff
@@ -66,6 +66,12 @@
 	var/hacked = 0 //If we have laws to destroy the station
 	var/datum/personal_crafting/handcrafting
 
+/datum/mob/living/simple_animal/drone/spec_life(mob/living/simple_animal/drone = D)
```
Comment (Super3222) [inline code/modules/mob/living/simple_animal/friendly/drone/_drone.dm:69]: It's still defined as `/datum/mob/living/simple/animal...` which I'm pretty sure won't work in the game...
You need `mob/living/simple_animal/drone/life()` instead.
The parameter is also not identified as a variable. You won't need it anyway.

**Comment by Ephemeralis** [comment]
One of these days you need to stop using the web editor and actually compile/test your changes before making PRs for them.

**Comment by Super3222** [comment]
> One of these days you need to stop using the web editor and actually compile/test your changes before making PRs for them.

I already told him that 20 PR's ago.

**Comment by ShadowDeath6** [comment]
Make clock drones die upon spawn and remove every other change in this pr :ok_hand:

**Comment by AsV9** [comment]
Fix & reopen.

Also y no drones on central command‽

**Pull request closed without merging**